docs/en/observability/categorize-logs.asciidoc

```diff
@@ -6,53 +6,55 @@ log messages are the same or very similar, so classifying them can reduce
 millions of log lines into just a few categories.
 
 Within the {logs-app}, the *Categories* page enables you to identify patterns in
-your log events quickly. Instead of manually identifying similar logs, the logs categorization view lists
-log events that have been grouped based on their messages and formats so that you can take action
-quicker.
+your log events quickly. Instead of manually identifying similar logs, the logs 
+categorization view lists log events that have been grouped based on their 
+messages and formats so that you can take action quicker.
+
 
 [[create-log-categories]]
 == Create log categories
 
-Create a machine learning job to categorize log messages automatically. Machine learning will observe
-the static parts of the message, cluster similar messages, and classify them into message categories.
+Create a 
+{ml-docs}/ml-configuring-categories.html[{ml} job to categorize log messages] 
+automatically. {ml-cap} observes the static parts of the message, cluster 
+similar messages, classify them into message categories, and detect unusually 
+high message counts in the categories. 
+
+[role="screenshot"]
+image::images/log-create-categorization-job.jpg[Configure log categorization job]
+
+1. Select *Categories*, and you are prompted to use {ml}, which carries out the 
+   log rate categorizations.
+2. Choose a time range for the {ml} analysis. By default, the {ml} job analizes 
+   log messages no lder than four weeks and continues indefinitely.
+3. Add the indices that contain the logs you want to analyze.
+4. Click *Create ML job*. The job is created and it starts run. It takes a few 
+   minutes for the {ml} robots to collect the necessary data. After the job 
+   processed the data, you can view the results.
 
-1. Select *Categories*, and you'll be prompted to use machine learning, which will carry out the log rate categorizations.
-2. Choose a time range for the machine learning analysis.
-3. Add the Indices that contain the logs you want to analyze.
-4. Click *Create ML job*.
-5. You're now ready to explore your log categories.
 
 [[analyze-log-categories]]
 == Analyze log categories
 
-Here we see the *Categories* page, which lists various log categories from
-`elasticsearch.server` logs, along with the maximum anomaly score on the right.
+The *Categories* page lists all the log categories from the selected indices. 
+You can filter the categories by indices. The screenshot below shows the 
+categories from the `elastic.agent` log.
 
 [role="screenshot"]
-image::images/log-categories.png[Log categories]
+image::images/log-categories.jpg[Log categories]
```

Comment:
Should this not be `.png`?

Reply:
I pushed a log-categories.jpg in this PR along with the textual changes, so the extension of this particular image is correct here.